docs/tables/github_repository_sbom.md

```diff
@@ -0,0 +1,22 @@
+# Table: github_repository_sbom
+
+The `github_repository_sbom` table can be used to query information about packages listed in the SBOM of a repository.
+
+**You must specify which repository** in the where or join clause using the `repository_full_name` column.
+
+## Examples
```

Comment:
Are there any other examples you'd want to include based on how you'd use the table to view SBOM/package information? For instance, is there anything in `packages` you'd want to validate specifically?